recipes/numcosmo/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set major_minor = "0.14" %}
```

Comment:
Can you make this just a single version statement,  that makes it easier for our tick bot to automatically keep this feedstock up to date.

Reply:
It's done. Some error came up but it should fixed now.